public/gameInfo.php

```diff
@@ -253,7 +253,41 @@ class='break-words whitespace-normal leading-normal'
 );
 ?>
 <?php if ($isFullyFeaturedGame): ?>
-    <?php RenderOpenGraphMetadata($pageTitle, "game", media_asset($gameData['ImageIcon']), "Game Info for $gameTitle ($consoleName)"); ?>
+    <?php
+        function generateMetaDescription(
+            string $gameTitle,
+            string $consoleName,
+            int $numAchievements = 0,
+            int $gamePoints = 0,
+            int $gameRetroPoints = 0,
+            bool $isEventGame = false,
+        ): string {
+            if ($isEventGame) {
+                return "$gameTitle: An event at RetroAchievements. Check out the page for more details on this unique challenge.";
+            } elseif ($numAchievements === 0 || $gamePoints === 0) {
+                return "No achievements have been created yet for $gameTitle. Join RetroAchievements to request achievements for $gameTitle and earn achievements on many other classic games.";
+            }
+
+            $localizedPoints = localized_number($gamePoints);
+            $localizedRetroPoints = localized_number($gameRetroPoints);
+
+            return "There are $numAchievements worth $localizedPoints ($localizedRetroPoints) points. $gameTitle for $consoleName - explore and compete on this classic game at RetroAchievements.";
```

Comment:
> "There are 78 achievements worth 800 (5,944) points. Super Mario Bros. for NES - explore and compete on this classic game at RetroAchievements."

"achievements" is missing.

```
<meta property="og:description" content="There are 78 worth 800 (5,931) points. Super Mario Bros. for NES - explore and compete on this classic game at RetroAchievements.">
```

Reply:
Good catch with the "achievements" being missing -- I think I stumbled due to the variable name. I also agree with not including RetroPoints and have removed it as of the latest commit.